engine/cld/config/env/config.go

```diff
@@ -208,6 +218,8 @@ var (
 		"onchain.aptos.deployer_key":                              {"ONCHAIN_APTOS_DEPLOYER_KEY", "APTOS_DEPLOYER_KEY"},
 		"onchain.tron.deployer_key":                               {"ONCHAIN_TRON_DEPLOYER_KEY", "TRON_DEPLOYER_KEY"},
 		"onchain.sui.deployer_key":                                {"ONCHAIN_SUI_DEPLOYER_KEY"},
+		"onchain.ton.deployer_key":                                {"ONCHAIN_TON_DEPLOYER_KEY", "TON_DEPLOYER_KEY"},
+		"onchain.ton.wallet_version":                              {"ONCHAIN_TON_WALLET_VERSION", "TON_WALLET_VERSION"},
```

Comment:
The second string in the slice is the alternative env var key to support backwards compatibility. Since this is a new chain, I would suggest removing it. This is similar to the SUI entry slightly above

```suggestion
		"onchain.ton.deployer_key":                                {"ONCHAIN_TON_DEPLOYER_KEY"},
		"onchain.ton.wallet_version":                              {"ONCHAIN_TON_WALLET_VERSION"},
```

Reply:
Suggestion applied in the latest commit